### PR TITLE
feat(route): add juejin news

### PR DESCRIPTION
**Pull request by CancerGary**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/juejin/news/739332228916791
```

## 新RSS检查列表 / New RSS Script Checklist

- [x] 这个PR中包含了新的路由吗? Does this PR add new route?
  - 如果有, 请完成检查列表. If yes, please finish the check list
  - **如果你的PR符合下方某个事项, 也请注明. If any of the checklist item meets your PR, please fill it out.**
  - [x] <- 这样打勾
- [x] 是否提供了文档? Documentation provided?
  - [ ] 是否提供了英文文档? EN Documentation provided?
- [x] 是否支持全文获取? Is this RSS Script support fulltext?
  - [x] 如果全文获取中需要访问文章链接, 是否使用了缓存? If fulltext requires to fetch detail pages, is cache used in the process?
  - [缓存说明](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun) | [How to use cache](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun)
- [ ] 目标是否有明显的反爬/频率限制? Is there any sign of anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? (延长缓存时间, 写文档说明, etc.) If yes, do your code reflect this sign? (e.g. write documentations, use long cache time)
- [x] 目标是否有提供日期？ Is there a date in the source?
  - [x] 如果有，包是否正确解析？ If there is, can this script provide this info?
  - [x] 如果有提供解析能力，时区是否正确调整？ Is the timezone correctly provided?
  - 如果有提供日期，但是没有提供解析，请说明原因 If there is a date but this script does not parse, please provide your reason.
- [ ] 是否引入的新的包? Any new package introduced?
  - 如果有, 请说明原因. If yes, please state your reason
- [ ] 是否使用了`Puppeteer`? Make use of `Puppeteer`?
  - 如果有, 请说明原因. If yes, please state your reason

### CODE REVIEW

File: lib/routes/juejin/utils.js [hunk 1]
```diff
@@ -27,20 +27,39 @@ async function load(link) {
...
 }
 
+function ParseArticleTypeItem(item) {
+    const pubdate = new Date(parseInt(item.article_info.ctime) * 1000 - 8 * 60 * 60 * 1000);
+    const link = `https://juejin.cn/post/${item.article_id}`;
+    // 列表上提取到的信息
+    const single = {
+        title: item.article_info.title,
+        description: `${(item.breif_content || '无描述').replace(/[\x00-\x08\x0b-\x0c\x0e-\x1f\x7f]/g, '')}`,
+        pubDate: pubdate.toUTCString(),
```
Comment (SettingDust) [inline lib/routes/juejin/utils.js:37]: 正确处理时间

File: lib/routes/juejin/utils.js [hunk 2]
```diff
@@ -27,20 +27,39 @@ async function load(link) {
     return { description };
 }
 
+function ParseArticleTypeItem(item) {
+    const pubdate = new Date(parseInt(item.article_info.ctime) * 1000 - 8 * 60 * 60 * 1000);
```
Comment (SettingDust) [inline lib/routes/juejin/utils.js:31]: 正确处理时间

File: lib/routes/juejin/utils.js [hunk 3]
```diff
@@ -27,20 +27,39 @@ async function load(link) {
...
+        description: `${(item.breif_content || '无描述').replace(/[\x00-\x08\x0b-\x0c\x0e-\x1f\x7f]/g, '')}`,
+        pubDate: pubdate.toUTCString(),
+        author: item.author_user_info.user_name,
+        link,
+    };
+    return { link, single };
+}
+
+function ParseContentTypeItem(item) {
+    const pubdate = new Date(parseInt(item.content_info.ctime) * 1000 - 8 * 60 * 60 * 1000);
```
Comment (SettingDust) [inline lib/routes/juejin/utils.js:45]: 正确处理时间

**Review by SettingDust — CHANGES_REQUESTED** [review]

File: lib/routes/juejin/utils.js (hunk 1 above)
Comment (CancerGary) [reply lib/routes/juejin/utils.js]: migrated to `@/utils/timezone`

**Review by SettingDust — CHANGES_REQUESTED** [review]
https://github.com/DIYgod/RSSHub/blob/916e83d3ef4b7f8a3597b15c9e905d63a418cbfe/lib/routes/juejin/utils.js#L66-L68
```
 const other = await caches.tryGet(link, () => load(link)); 
 // 合并解析后的结果集作为该篇文章最终的输出结果 
 return { ...single, other }; 
```

File: lib/routes/juejin/utils.js [hunk 4]
```diff
@@ -27,20 +28,38 @@ async function load(link) {
     return { description };
 }
 
+function ParseArticleTypeItem(item) {
+    const pubdate = timezone(new Date(parseInt(item.article_info.ctime) * 1000), +8);
```
Comment (TonyRL) [inline lib/routes/juejin/utils.js:32]: Please use parseDate() from `@/utils/parse-date`

File: lib/routes/juejin/utils.js [hunk 5]
```diff
@@ -27,20 +28,38 @@ async function load(link) {
...
+        description: `${(item.breif_content || '无描述').replace(/[\x00-\x08\x0b-\x0c\x0e-\x1f\x7f]/g, '')}`,
+        pubDate: pubdate,
+        author: item.author_user_info.user_name,
+        link,
+    };
+    return { link, single };
+}
+
+function ParseContentTypeItem(item) {
+    const pubdate = timezone(new Date(parseInt(item.content_info.ctime) * 1000), +8);
```
Comment (TonyRL) [inline lib/routes/juejin/utils.js:46]: Please use parseDate() from `@/utils/parse-date`

**Review by TonyRL — CHANGES_REQUESTED** [review]
v2 script standard https://github.com/DIYgod/RSSHub/discussions/8236

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

**Pull request merged**